scripts/xml_to_csv/xml_to_csv.py

```diff
@@ -0,0 +1,40 @@
+# Written By: Jodi Gunawan
+# Date: Nov 15th, 2018
+# Description:  This script creates csv file from xml input file under the file train_labels
+
+import os
+import glob
+import pandas as pd
+import xml.etree.ElementTree as ET
+
+
+def xml_to_csv(path):
+    xml_list = []
+    for xml_file in glob.glob(path + '/*.xml'):
+        tree = ET.parse(xml_file)
+        root = tree.getroot()
+        for member in root.findall('object'):
+            value = (root.find('filename').text,
+                     int(root.find('size')[0].text),
+                     int(root.find('size')[1].text),
+                     member[0].text,
+                     int(member[4][0].text),
+                     int(member[4][1].text),
+                     int(member[4][2].text),
+                     int(member[4][3].text)
+                     )
+            xml_list.append(value)
+    column_name = ['filename', 'width', 'height', 'class', 'xmin', 'ymin', 'xmax', 'ymax']
+    xml_df = pd.DataFrame(xml_list, columns=column_name)
+    return xml_df
+
+
+def main():
+    for directory in ['train_labels']:
+        image_path = os.path.join(os.getcwd(), 'images/train_labels/{}'.format(directory)) 
+        xml_df = xml_to_csv(image_path)
+        xml_df.to_csv('data/{}_converted.csv'.format(directory), index=None)
+        print('Successfully converted xml to csv.')
+
+
+main()
```

Comment:
A good python habit to get into is to include the if __name__ == 'main': definition in your file. The line just specifies that is the script is run by itself then it will just run the main() function. This will allow other classes to potentially import these functions without running the main() function by default, it's also just the python standard for scripts.